### PR TITLE
Issue #SB-5388 fix: Ui bug fixes

### PR DESCRIPTION
**Pull request by prajwal88**

### CODE REVIEW

**Comment by coolbung** [comment]
Please assign to @lakhanmandloi

**Comment by coolbung** [comment]
Please assign to @lakhanmandloi

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
 <div class="ui grid" *ngIf="!showExploreHeader">
-    <div class="center aligned twelve column row">
+    <div class="center aligned twelve column row" [class.header-height-before-login]="!userService.userid">
```
Comment (lakhanmandloi) [inline src/app/client/src/app/modules/core/components/main-header/main-header.component.html]: need to add `middle` class here

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
         <div class="one wide tablet computer only column"></div>
-        <div class="two wide tablet computer only column padding-top-15">
+        <div class="two wide tablet computer only column" [class.padding-top-5]="!userService.userid" [class.padding-top-15]="userService.userid">
```
Comment (lakhanmandloi) [inline src/app/client/src/app/modules/core/components/main-header/main-header.component.html]: need to remove `padding-top-5` class from here.

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html (hunk 1 above)
Comment (lakhanmandloi) [reply src/app/client/src/app/modules/core/components/main-header/main-header.component.html]: `middle` class should be placed before `aligned` class.

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html (hunk 2 above)
Comment (lakhanmandloi) [reply src/app/client/src/app/modules/core/components/main-header/main-header.component.html]: Both suggestions are based on following - 

header has 80px height.. while logo has max 40px height.. padding top 5 will not align it in middle.

**Pull request merged**